packages/alfa-rules/src/sia-er62/diagnostics.ts

```diff
@@ -49,7 +49,7 @@ export class ElementDistinguishable
   }
 
   public withStyle(
-    ...styles: ReadonlyArray<readonly [Name, string]>
+    styles: Iterable<readonly [Name, string]>
```

Comment:
Indeed, my bad… `...` accept any iterable when used as spread operator, not as rest operator (i.e. when using a variable, not when declaring a parameter) 🙈 

We should keep the rest operator here (and thus, the `ReadonlyArray`) in order to allow `.withStyle(["prop1", "value"], ["prop2", "value2"])` without needing to wrap everything in `[ ]` again.

Sorry about that 😞

Reply:
It's alright, I'll revert that change, I just thought it was easier to read even though it's annoying to wrap